### PR TITLE
[#3740] Update dotnet samples to target Net 6 - Authentication samples

### PR DESCRIPTION
**Pull request by erquirogasw**
Addresses # 3740
#minor

## Description
This PR updates the Authentication samples to target .NET6

## Specific Changes
- Updated the following samples to target .NET6
  - Bot-authentication
  - Bot-authentication-msgraph
- README files updated to target .NET6

## Testing
Bot-authentication working
image (85)

Bot-authentication-msgraph working
image (86)

Teams-auth working
image (87)

### CODE REVIEW

File: samples/csharp_dotnetcore/46.teams-auth/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This sample utilizes an app setting `UseSingleSignOn` to add `TeamsSSOTokenExcha
 ## Prerequisites
 
 - Microsoft Teams is installed and you have an account (not a guest account)
-- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [.NET SDK](https://dotnet.microsoft.com/download) version 6.0
```
Comment (sw-joelmut) [inline samples/csharp_dotnetcore/46.teams-auth/README.md]: This bot is already included in the PR https://github.com/southworks/BotBuilder-Samples/pull/396/files#diff-85f929f06830df2afd3359bebeea0b190f11ad45a24486a49d906fb3074ab5d0

**Review by sw-joelmut — CHANGES_REQUESTED** [review]

File: samples/csharp_dotnetcore/46.teams-auth/README.md (hunk 1 above)
Comment (erquirogasw) [reply samples/csharp_dotnetcore/46.teams-auth/README.md]: Fixed. Thank you!

Comment (sw-joelmut) [reply samples/csharp_dotnetcore/46.teams-auth/README.md]: Remove it from the description as well

**Review by ceciliaavila — APPROVED** [review]

**Pull request closed without merging**